alacritty_colorscheme/cli.py

```diff
@@ -182,6 +188,17 @@ def replace_colorscheme(colors_path: str, config_path: str, colorscheme: str, ba
             )
             vimrc_background_file.write(vimrc_background_content)
 
+def get_applicable_colorscheme(colorschemes, colorscheme, reverse_toggle):
+    if colorscheme in colorschemes:
+        realindex = colorschemes.index(colorscheme)
+        inc = len(colorschemes) - 1 if reverse_toggle else 1
```

Comment:
```suggestion
        inc = - 1 if reverse_toggle else 1
```

Reply:
Ah yes, you're right. I'm used to C's modolo operator.